codec-http/src/test/java/io/netty/handler/codec/http/HttpUtilTest.java

```diff
@@ -118,4 +120,82 @@ public void testDoubleChunkedHeader() {
         List<String> expected = Collections.singletonList("chunked");
         assertEquals(expected, message.headers().getAll(HttpHeaderNames.TRANSFER_ENCODING));
     }
+
+    private static List<String> allPossibleCasesOfContinue() {
+        final List<String> cases = new ArrayList<String>();
+        final String c = "continue";
+        for (int i = 0; i < Math.pow(2, c.length()); i++) {
+            final StringBuilder sb = new StringBuilder(c.length());
```

Comment:
Can we keep it this way, it's a unit test and I like keeping them straightforward? :)